src/css/App.css

```diff
@@ -523,18 +523,28 @@ p {
 
 .under-header-content {
   background-color: #0a0a23;
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  align-items: center;
   z-index: 1;
   position: absolute;
   width: 100%;
-  height: 50px;
   top: 38px;
   text-align: center;
 }
 
+.title-wrapper {
+  height: 50px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.offline-banner {
+  height: 38px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: lightblue;
+}
```

Comment:
The styles were copied from the OfflineWarning component over the main repo https://github.com/freeCodeCamp/freeCodeCamp/blob/0c0611229e23e20000bd287736fa9dc36566f7b1/client/src/components/OfflineWarning/offline-warning.css